components/sysutils/exfat/patches/configure.ac.patch

```diff
@@ -0,0 +1,22 @@
+--- tries/configure.ac.orig	2018-09-15 07:03:24.000000000 +0000
```

Comment:
Can you make a comment here to describe the reason why this patch exists so we can make decisions if we want to keep it or not in the future?